src/chrome/locale/en/https-everywhere.dtd

```diff
@@ -8,7 +8,7 @@
 <!ENTITY https-everywhere.menu.donate_eff_imperative "Donate to EFF">
 <!ENTITY https-everywhere.menu.observatory "SSL Observatory Preferences">
 <!ENTITY https-everywhere.menu.globalEnable "Enable HTTPS Everywhere">
-<!ENTITY https-everywhere.menu.blockUnencryptedRequests "Block all unencrypted requests">
+<!ENTITY https-everywhere.menu.blockUnencryptedRequests "Encrypt All Sites Eligible">
```

Comment:
I might also add " (EASE)" to this, to make it easy (no pun intended) to refer to.